config/api.yml

```diff
@@ -468,7 +468,7 @@
     :identifier: cloud_volume
     :options:
     - :collection
-    :verbs: *gp
+    :verbs: *gpppd
```

Comment:
I think you only need `gpd` here....

Since you're adding the 'd' - can you add tests for deleting via DELETE using the collection/member routes, and that RBAC is enforced for both of these too? I have a feeling this may require some more configuration (i.e. adding an explicit entry for delete with the appropriate identifier for each)